extensions/common/gcp/gcp-core/src/main/java/org/eclipse/edc/gcp/iam/CredentialsUtil.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (c) 2024 Google LLC
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Google LLC - Initial implementation
+ *
+ */
+
+package org.eclipse.edc.gcp.iam;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import org.eclipse.edc.gcp.common.GcpServiceAccount;
+
+/**
+ * Interface for credentials providing access tokens.
+ */
+public interface CredentialsUtil {
```

Comment:
please avoid the `Util` word that could lead to undefined scope and "trash bin classes", `CredentialsManager` could be a more appropriate name

Reply:
Done